applications/conference/src/conf_config_req.erl

```diff
@@ -52,9 +49,53 @@ fetch_config(JObj, ConfigName, Profile) ->
             wh_util:log_stacktrace(ST)
     end.
 
+-spec default_profile() -> wh_json:object().
+default_profile() ->
+    whapps_config:get(?CONFIG_CAT
+                      ,[<<"profiles">>, ?DEFAULT_PROFILE_NAME]
+                      ,wh_json:from_list(?DEFAULT_PROFILE_CONFIG)
+                     ).
+
+-spec profiles(ne_binary(), wh_json:object()) -> wh_json:object().
+profiles(ConfigName, Profile) ->
+    NewContent = add_conference_params(ConfigName, Profile),
+    wh_json:from_list([{ConfigName, NewContent}]).
+
+-spec add_conference_params(ne_binary(), wh_json:object()) -> wh_json:object().
+add_conference_params(ConfigName, Profile) ->
+    Conference = get_conference(ConfigName),
+    Props = props:filter_undefined(
+              [{<<"max-members">>, max_participants(Conference)}
+              %% ,{<<"max-members-sound">>, <<"tone_stream://v=-7;>=2;+=.1;%(377,7,577,770);v=-7;>=3;+=.1;%(877,0,377,477)">>}
```

Comment:
include this, it is needed by freeswitch to play in case conference is full. try to use the same as set in conference whapp